code/modules/RCD/tile painter.dm

```diff
@@ -10,8 +10,23 @@
 
 	sparky				= 0
 
+	var/emagged 		= 0
+
 	schematics = list(/datum/rcd_schematic/clear_decals)
 
 /obj/item/device/rcd/tile_painter/New()
-	schematics += typesof(/datum/rcd_schematic/tile)
+	schematics += typesof(/datum/rcd_schematic/tile) - /datum/rcd_schematic/tile/emagged
```

Comment:
it might be more futureproof to make this `- typesof(/datum/rcd_schematic/tile/emagged)` in case someone ever wants to add more emagged types, but I guess that's not a big deal - keep in mind for future